Gemfile

```diff
@@ -21,9 +21,8 @@ rails_version = ENV["RAILS_VERSION"] || "default"
 
 rails = case rails_version
   when "default"
-    ">= 3.1.0"
```

Comment:
Good point. I removed it. Although it won't help you guys if gemspeck has railties 4.1 mentioned?

Dummy application tests wont work with rails 3.x anymore. I guess the gem otherwise works still with the old one. hmm.